<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverviewPage</class>
 <widget class="QWidget" name="OverviewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>998</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>471</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>87</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 15px; font-weight: bold</string>
           </property>
           <property name="text">
            <string>Balances</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelWalletStatus">
           <property name="toolTip">
            <string>The displayed information may be out of date. Your wallet automatically synchronizes with the Dash network after a connection is established, but this process has not completed yet.</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 15px; color:rgb(255, 15, 15); </string>
           </property>
           <property name="text">
            <string notr="true">(out of sync)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="spacing">
          <number>12</number>
         </property>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="labelWatchAvailable">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Your current balance in watch-only addresses</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 15px</string>
           </property>
           <property name="text">
            <string notr="true">0.000 000 00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="labelImmature">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Mined balance that has not yet matured</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 15px</string>
           </property>
           <property name="text">
            <string notr="true">0.000 000 00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="4">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="labelWatchImmature">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Mined balance in watch-only addresses that has not yet matured</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 15px</string>
           </property>
           <property name="text">
            <string notr="true">0.000 000 00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labelTotalText">
           <property name="text">
            <string>Total:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelImmatureText">
           <property name="text">
            <string>Immature:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelBalanceText">
           <property name="text">
            <string>Available:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="labelTotal">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Your current total balance</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 25px</string>
           </property>
           <property name="text">
            <string notr="true">0.000 000 00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelSpendable">
           <property name="text">
            <string>Spendable:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="labelWatchonly">
           <property name="text">
            <string>Watch-only:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QLabel" name="labelWatchTotal">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Current total balance in watch-only addresses</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 25px</string>
           </property>
           <property name="text">
            <string notr="true">0.000 000 00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelBalance">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Your current spendable balance</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 15px</string>
           </property>
           <property name="text">
            <string notr="true">0.000 000 00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="3">
          <widget class="Line" name="lineWatchBalance">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="Line" name="lineSpendableBalance">
           <property name="styleSheet">
            <string notr="true">background-color: transparent</string>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>370</y>
        <width>231</width>
        <height>121</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QWidget" name="widgetRecent" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>441</width>
        <height>181</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>225</height>
       </size>
      </property>
      <widget class="QFrame" name="frameBlockchainInfoData_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>331</width>
         <height>171</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: transparent;
; font-size: 15px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_CurrentBlock_value_3">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_count_4">
          <property name="text">
           <string>Node Count:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_Nethash_value_3">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="label_count_2">
          <property name="styleSheet">
           <string notr="true">;</string>
          </property>
          <property name="text">
           <string>Please Wait...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_Algo_3">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Algorithm:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_11">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_CurrentBlock_2">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Current block:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="countLabelDIP3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="labelMN_2">
          <property name="text">
           <string>Peers:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_Nethash_3">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Difficulty:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_Algo_2">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>X25X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="10" column="0">
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="12" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
